### PR TITLE
Adding windows binary option

### PR DESCRIPTION
**Pull request by maritsandstad**
- [x] Tests added
- [x] Documentation added
- [x] Example added (in the documentation, to an existing notebook, or in a new notebook)
- [x] Description in ``CHANGELOG.rst`` added (single line such as: ``(`#XX <https://github.com/openscm/openscm-runner/pull/XX>`_) Added feature which does something``)

(Not completely sure how much of this checklist needs to be here....)

### CODE REVIEW

**Comment by maritsandstad** [comment]
Hi @znicholls, here's an attempt at adding a windows binary for ciceroscm

**Comment by znicholls** [comment]
> Not completely sure how much of this checklist needs to be here

Tests and changelog

Documentation and example you can just tick

File: src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py [hunk 1]
```diff
@@ -55,8 +56,12 @@ def get_version(cls):
         OSError
             The CICERO-SCM binary cannot be run on the operating system
         """
+        if platform.system() == "Windows":
```
Comment (znicholls) [inline src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py:59]: Can we use `get_executable` for this rather than duplicating the code?

**Review by znicholls — COMMENTED** [review]
https://github.com/maritsandstad/openscm-runner/pull/16 turns on the windows tests in CI, let's see what chaos ensues

File: src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py (hunk 1 above)
Comment (maritsandstad) [reply src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py]: Done

**Comment by maritsandstad** [comment]
Ok, so after digging out some really moronic mistakes of mine, it now sort of works, but still sort of doesn't... The issue now seems to be that since, windows had double / in the file paths, the really long and annoying names that I used to cut so they would work with the fortran code, now don't anymore because just being on windows tipped the path over. Solvable enough I guess, just so frustrating...

**Comment by maritsandstad** [comment]
I think now the main part is updating the expected results for the windows run, not entirely sure how to do that from my end... Any tips @znicholls ?

**Comment by znicholls** [comment]
> I think now the main part is updating the expected results for the windows run, not entirely sure how to do that from my end... Any tips @znicholls ?

The results should be the same, independent of operating system right?

**Comment by znicholls** [comment]
If you run it locally on a windows machine, do the tests pass?

**Comment by maritsandstad** [comment]
At the moment I'm not really set up to do that. Would prefer not to have to spend a day getting that running...

**Comment by maritsandstad** [comment]
Hi @znicholls, I fixed the windows binary I think... It would be great if this could be merged in...

**Comment by znicholls** [comment]
Looks good. Just CHANGELOG left to do then we're good.

As a nit, can you also use the checklist at the top of the pull request please. You can tick things that aren't relevant (so for this merge request an example can just be ticked as we don't need it). Having everything ticked helps me see that everything has actually been done.

**Comment by maritsandstad** [comment]
Ok, added an entry to the CHANGELOG, hopefully it's correct, I tend to always muck that up... Also, sorry about not ticking the boxes, I know it's not the first time you've told me. I guess I just have a problem telling when the various things are actually at a box-ticking level...

**Comment by znicholls** [comment]
Thanks. I tweaked the CHANGELOG in https://github.com/maritsandstad/openscm-runner/pull/17/files

In short, you don't need to create a new section if the section your change belongs in is already there, you can just add your change to the list of changes. Hope that makes sense

**Comment by maritsandstad** [comment]
Great, thank you, I guess I understood that. Can't guarantee I'll retain the information until I need it the next time...

**Review by znicholls — APPROVED** [review]

**Comment by znicholls** [comment]
All good, nice work

**Pull request merged**